code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -809,3 +809,17 @@
 			target.say("*scream")
 			spawn(10)
 			user.visible_message("<span class='warning'>[user] stops the [src] and leaves [target] resting as they are.</span>", "<span class='warning'>You turn the [src] off and let go of [target].</span>")
+
+/obj/machinery/reagentgrinder/verb/Eject()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject contents"
+
+	if (!ishuman(usr))
+		return
+		usr.visible_message(

```

Comment:
This line is unreachable. Looks like skull's comment was marked resolved but the issue was not actually resolved.